### ISSUE TITLE
Can't use biometric authentication even though the phone has a fingerprint sensor

### ISSUE THREAD

**Issue by Linerly** [element-hq/element-android#7064]
### Steps to reproduce

1. Enable biometric authentication in Settings

#### What did you expect?
Biometrics authentication to work, like other apps

#### What happened instead?
![Screenshot_2022-09-07-21-00-38-200](https://user-images.githubusercontent.com/47475676/188898094-887c77ab-bf04-4a98-b9c0-bbb55acf86d6.jpg)
![Screenshot_2022-09-07-21-01-21-454](https://user-images.githubusercontent.com/47475676/188898112-b87eb994-841e-434f-8f97-a664d47a640c.jpg)

### Your phone model

SHARP Aquos V6

### Operating system version

Android 12

### Application version and app store

1.4.34 [40104340] (F-ba6f7c42)

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
I've only seen this kind of behaviour while testing it on the emulator, where the implementation of fingerprint emulation is a bit buggy. Could you send some logs or a rageshake after you've tried enabling biometric auth and reference this issue (#7064)?

**Comment by Linerly** [comment]
I have sent the logs when it happened, with the same title as this issue

**Comment by jmartinesp** [comment]
The error right after enabling biometric auth and creating the biometric key is:

```
android.security.keystore.UserNotAuthenticatedException: User not authenticated
```

We might need to tweak the biometric key creation, as I'm beginning to think that the expected default parameters might not be the same as the docs mention for all OS versions and vendors.

**Comment by AlphaDE** [comment]
I have the same issue with a Pixel 7 Pro on Android 13, biometric gets suddenly deactivated and cannot be reenabled later again.
Might be related to the fact, that sometimes when for any reason there are too many bad attempts (phone in pocket), the sensor gets deactivated by the system and Element misinterprets that?

**Comment by PracticalChip** [comment]
Same issue here also on pixel 7 pro

**Comment by losetsinn** [comment]
Same issue here Pixel 7.
Biometric geht deactivatet and can not reenebeld. I can Push the Slider top enabel but a Bubbles came with "Can't activate" gut the Slider stay in enable.

If i know Lock the Phone the biometric works in Element again.

I think also thats a Problem with bad attemps is. But Element are to sensitive. Others Apps works without problems.

**Comment by jordanbtucker** [comment]
Same issue on my Pixel 6a. It seemed to work for the first day, but now it won't stay enabled. When I enable it, I get a toast saying it couldn't be enabled (same as in the first screen shot).

And then when I close and reopen the app, I get same message in the screenshot in the comment before mine.

**Comment by AlphaDE** [comment]
Any chance to see this fixed? It's long standing...

**Comment by losetsinn** [comment]
I think IT is a counter Problem. One fail biometric in a other App and element deactivate biometric. 
Only Lock and unlock the Pixel with biometric solve the Problem.

**Comment by AlphaDE** [comment]
On my Pixel 7 Pro, the app crashes when even trying to enter access control after entering the pin. No chance to get there anymore.

**Comment by hardwareadictos** [comment]
Already reported there: https://github.com/vector-im/element-android/issues/8649

So this is opened from a year ago and it's still not fixed? 😨

**Comment by Sev3rin** [comment]
Right, and it's the reason why no one in my circle of friends uses Matrix. App alternatives don't exist and that's why they switched to alternatives. Most of ten uses Googlephones.

**Comment by hardwareadictos** [comment]
> Right, and it's the reason why no one in my circle of friends uses Matrix. App alternatives don't exist and that's why they switched to alternatives. Most of ten uses Googlephones.

That's not great. Can someone on the dev team take a look at this? Or at least give an answer to know that's being revised?

**Comment by jordanbtucker** [comment]
@Sev3rin App alternatives don't exist?

https://matrix.org/ecosystem/clients/

**Comment by Sev3rin** [comment]
> @Sev3rin App alternatives don't exist?
> 
> https://matrix.org/ecosystem/clients/

Thanks für Posting the alternatives,

Only If you test the alternatives for Android they have No Lock with biometric funktion or they are forks of Elements with the same issue.

🙄

**Comment by simmstein** [comment]
Same issue on samsung n20.

**Comment by oldgithubman** [comment]
Same issue on pixel 7 pro android 16
